mxd/README.md

```diff
@@ -349,17 +477,191 @@ restriction. Those informations will be used for negotiating a contract with the
+
+
+```shell
+curl --location 'http://localhost/bob/management/v2/transferprocesses/<ID>' \
+--header 'X-Api-Key: password' | jq
+```
+
+- If the transfer was **successful**, Bob will see an ouput as shown below.
+- If the transfer was **unsuccessful**, the transfer state will be `TERMINATED`.
+```json
+{
+  "@id": "6d6bca4e-4da5-4ed3-9fe5-2b98623d9a59",
+  "@type": "edc:TransferProcess",
+  "edc:correlationId": "6d6bca4e-4da5-4ed3-9fe5-2b98623d9a59",
+  "edc:state": "STARTED",
+  "edc:stateTimestamp": 1702990026966,
+  "edc:type": "CONSUMER",
+  "edc:assetId": "3",
+  "edc:contractId": "Mw==:Mw==:NmY5MDA4OGEtOWY1ZC00YmYyLWFiZjMtMjRiNzY0YzEyOTk4",
+  "edc:callbackAddresses": [],
+  "edc:dataDestination": {
+    "@type": "edc:DataAddress",
+    "edc:type": "HttpProxy"
+  },
+  "edc:connectorId": "BPNL000000000001",
+  "@context": {
+    "dct": "https://purl.org/dc/terms/",
+    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "dcat": "https://www.w3.org/ns/dcat/",
+    "odrl": "http://www.w3.org/ns/odrl/2/",
+    "dspace": "https://w3id.org/dspace/v0.8/"
+  }
+}
+```
+
+The response shows him that he has a `STARTED` transfer with the type `CONSUMER` of the asset with the ID `3`.
+Note, that a consumer pull transfer will **not** advance to the `COMPLETED` state, but instead will stay in state `STARTED`.
+This means, that Bob can now request the data using the information received by his backend.
+
+```shell
+curl --location 'http://localhost/bob/management/edrs?assetId=3' \
+--header 'X-Api-Key: password' | jq
+```
+
+### 5.3 Consume the data
+
+In his backend, Bob can now see the following output:
+```json
+{
+  "id": "841e3cd7-add0-47fd-adef-ea8074ec50af",
+  "endpoint": "http://alice-tractusx-connector-dataplane:8081/api/public",
+  "authKey": "Authorization",
+  "authCode": "eyJhbGciOiJFUzI1NiJ9.eyJleHAiOjE3MDI5OTE2NTUsImRhZCI6InIyaVpYUE9kSGJvUHBMTXZTS1hlSjgrc05WSThuS0V6WnhHa3hrSHN0YVJ1Z3l3TTlNRkxQbWJaT0toOUFXKzRwMlVZT3hEQitGMlNsbEFnTGZ0c1IyVVIxbE5RZEhNeFVDYkM3Nm12U0xmdkw4RFpZM2E0bG1pTXU4VUU3Zk92aXczaGlyKy9zWFRtNDU5M3EweXBHRmRaZ1VORko3ZnFRbjI2OStyWmZzWnZtaitLaWZZMURJcEQ5ZEQ1NkNURW1naTczU1VyRWF5RWtlc3dTWTJRZEluNWxOSFVReDdJMWNkZFZJdXkyZkMzYVJVWENqK003MjVmZ2ZERkwwR2N0ZTFkSWVjOUhHYTRyVW5WL0ZWbVJnUURUZSsza0thaGQ3WjVDTE5wcVVtREx5SGpBNHV3U1dkWHZ3emxDeGlxRzZ4eDg0amZtVDZlMzNLeG9NT1RVUUpnbWZyQWtJUkJ5a0dPa3BvN2p1a2dWVWk1bGNyUVEyMjQxYW5xa3ZzbWh5cnN1dUdTcnBxdEl4ZzQrd1lSSDYvTnBvRUUvemRxS0xLaE9hcz0iLCJjaWQiOiJNdz09Ok13PT06Tm1ZNU1EQTRPR0V0T1dZMVpDMDBZbVl5TFdGaVpqTXRNalJpTnpZMFl6RXlPVGs0In0.RvbdPWKIrBmMqIN7kvPnwoAOJMug-VKfREKrHWPOmfp0VGSsVuma7SAJaMb9cPT7X4lNZilw36GYWDh92FHSVA",
+  "properties": {}
+}
+```
+Bob can now request the data using the provided information as follows:
+```shell
+curl -X GET -H '<authKey>: <authCode>' <endpoint>
+```
+
+
+> In this example, we can not use the endpoint URL as is, because we are working with a local Kubernetes cluster and can not use the cluster internal URL.  
```

Comment:
```suggestion
> In this example, we can not use the endpoint URL as is, because we are working with a local Kubernetes cluster and can not use the cluster internal URL.
```